core/src/log.cpp

```diff
@@ -139,6 +147,33 @@ auto Logger::level() -> Logger::Level
     return state().level;
 }
 
+bool Logger::setFilePath(const std::string& filePath)
+{
+    if (!FileSystem::mount("/logs", std::make_unique<cubos::core::data::StandardArchive>("./logs", /*isDirectory=*/true,
+                                                                                         /*readOnly=*/false)))
+    {
+        return false;
+    }
+
+    auto fullPath = "/logs/" + filePath;
+    auto file = FileSystem::create(fullPath, /*directory=*/false);
+    if (file == nullptr)
+    {
+        return false;
+    }
```

Comment:
Here, you should only create the file if it doesn't already exist.

Reply:
Yeah, but `create` returns the file if alr exists (https://gamedevtecnico.github.io/cubos/docs/classcubos_1_1core_1_1data_1_1FileSystem.html#a4a5c6b0cb8c33de83caa6cdaf4147f87). I thought this way would be cleaner instead of chaining
```
get file
if file does not exist
   create
       if create fails
            return false
```

Reply:
Whoops, forgot about that property, mb.